src/Components/CookieBanner.js

```diff
@@ -18,6 +18,8 @@ export default function CookieBanner() {
       show={bannerVisibility}
       onHide={() => switchBannerMode()}
       scrollable
+      aria-label="cookie-banner"
```

Comment:
```
aria-label={I18n.t(`CookieBanner.label`, { ns: "cookieBanner" })}
```

Translation `de`
```
"Diese Webseite verwendet Cookies"
```

Translation `de`
```
"This website uses cookies"
```

Reply:
`en.json` and `de.json` are identical files. Probably used at the begging as a default with German content, dunno. I would remove it - what do you think?

Reply:
Nope, please provide English translation in a separate pull request. Or I can do it, if you think that's better.

Reply:
https://github.com/infopark-customers/scrivito-another-cookie-banner/pull/2